test/integration/model/find.test.js

```diff
@@ -1024,18 +1024,33 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         username: Sequelize.STRING
       }, { paranoid: true });
 
-      return User.sync({ force: true }).then(() => {
-        return User.bulkCreate([
-          {username: 'Bob'},
-          {username: 'Tobi'}
-        ]);
-      }).then(() => {
-        return User.destroy({ where: {username: 'Tobi'} });
-      }).delay(5000).then(() => {
-        return User.findAll({ paranoid: true });
-      }).then(users => {
-        expect(users.length).to.be.eql(1);
-      });
+
+      if (dialect === 'sqlite') 
```

Comment:
Will do on braces.

On SQLite:
The only difference is a `.delay()` after deleting a record. There appears to be some sort of race condition when deleting the record (https://travis-ci.org/sequelize/sequelize/builds/245954861) and immediately querying it back, which is what I was doing in my test. Only saw it on SQLite, but if I created 2 records, soft delete 1, and then `findAll`, I would get both records back. A brief `delay` seems to fix on SQLite, but sometimes caused an `Error: Expected 0 running queries. 1 queries still running in [POSTGRES] Model constructor should allow the user to specify indexes in options`, especially on Postgres test - thus only tested it on SQLite. It may be worth extending the delay some to get a more reliable result, but 500 seemed okay for my test.